packages/adapters/gupshup-whatsapp/src/GupShupWhatsappAdapter.ts

```
@@ -89,15 +90,17 @@ export class GupshupWhatsappProvider implements XMessageProvider {
 
   private readonly providerConfig?: IGSWhatsappConfig;
   private userHistory: XMessage[];
+  private callback: ((userHistory: XMessage[]) => boolean) | undefined;
 
   constructor(config?: IGSWhatsappConfig, userHistory: XMessage[] = []) {
     this.providerConfig = config;
     this.userHistory = userHistory;
+    this.callback = config?.callback;
   }
 
   private getMessageState = (eventType: String): MessageState => {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)